### PR TITLE
move docker files back to root

### PR DESCRIPTION
**Pull request by chrissng**
- reasons as states in https://github.com/datagovsg/airflow-pipeline/issues/23
- intention of putting them in `docker/` was to keep the root directory tidy

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,13 +4,13 @@ python:
...
+- docker build -t latest --no-cache .
 script:
 - docker-compose -f tests/docker-compose.test.yml up
 after_success:
 - docker login -u $DOCKER_USER -p $DOCKER_PASS
 - export REPO=$TRAVIS_REPO_SLUG
 - export TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH ; fi`
 - echo $REPO:$TAG
-- docker build -f docker/Dockerfile -t $REPO:$TAG --no-cache .
+- docker build -f Dockerfile -t $REPO:$TAG --no-cache .
```
Comment (lawliet89) [inline .travis.yml]: Can remove the `-f` flag.

**Review by lawliet89 — APPROVED** [review]
:1st_place_medal:

**Comment by lawliet89** [comment]
I don't have permissions to merge though.

**Comment by chrissng** [comment]
oops! okay granted!

**Pull request merged**